queries/cedar/highlights.scm

```diff
@@ -0,0 +1,69 @@
+; Function calls
+(call_expression
+  function: (selector_expression
+    field: (field_identifier) @function.method))
+
+[
+  "contains"
+  "containsAll"
+] @function
+
+; Annotations
+(annotation) @attribute
+
+; Identifiers
+[
+  (entity)
+  (identifier)
+] @variable
+
+[
+  (principal)
+  (action)
+  (resource)
+  (context)
+] @variable.builtin
```

Comment:
These two variable queries should be the very first to prevent overriding things like `(annotation)`